web/basic/header.html

```diff
@@ -190,6 +191,15 @@
                 <img class="c-sidebar-brand-minimized" src="img/tx.png" width="30" height="30" alt="txAdmin Logo">
             </a>
             <ul class="c-header-nav ml-auto mr-4">
+                <a class="c-header-nav-item">
+                    <svg class="c-sidebar-nav-icon" id="darkToggleLight">
+                        <use xlink:href="img/coreui_icons-full.svg#cil-sun"></use>
```

Comment:
There is no longer a need for the xlink namespace in SVG 2 namespace, it is deprecated. You can just use `href` in its place.

See [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href)